x-pack/plugins/reporting/public/management/report_listing.tsx

```diff
@@ -204,10 +204,6 @@ class ReportListingUi extends Component<Props, State> {
           throw error;
         }
       }
-
-      // Since the contents of the table have changed, we must reset the pagination
-      // and re-fetch. Otherwise, the Nth page we are on could be empty of jobs.
-      this.setState(() => ({ page: 0 }), this.fetchJobs);
```

Comment:
Without refresh after delete, this `fetchJobs` tends to predicable stale report artifacts for recently deleted reoprts  in the list.